src/js/lib/Scrollspy.jsx

```diff
@@ -141,6 +141,7 @@ export class Scrollspy extends React.Component {
 
   render () {
     let counter = 0
+    var styles = _.extend({},this.props.style)
```

Comment:
@maxkferg `_` is not defined. This library is not dependent on underscore or lodash.
Please use `Object.assign()` instead of `_`.